times_reader/main.py

```diff
@@ -199,28 +199,37 @@ def compare(
     if len(missing) > 0:
         print(f"WARNING: Missing {len(missing)} tables: {missing_str}")
 
+    additional_tables = set(data.keys()) - set(ground_truth.keys())
+    additional_str = ", ".join(
+        [f"{x} ({data[x].shape[0]})" for x in sorted(additional_tables)]
+    )
+    if len(additional_tables) > 0:
+        print(f"WARNING: {len(additional_tables)} additional tables: {additional_str}")
+    # Additional rows starts as the sum of lengths of additional tables produced
+    total_additional_rows = sum(len(data[x]) for x in additional_tables)
```

Comment:
@siddharth-krishna, what will this result in if `additional_tables` is empty?

Reply:
`sum([]) == 0` :)